app/Svg/Parser.hs

```diff
@@ -71,10 +71,19 @@ performParse :: T.Text -- ^ The title of the graph.
              -> String -- ^ The filename of the file that will be parsed.
              -> SqlPersistM ()
 performParse graphName inputFilename = do
+    deleteExistingGraph graphName
```

Comment:
This is good, but make sure to add a note to the documentation of this function to describe the new behaviour.